cli/src/test/java/com/devonfw/tools/ide/commandlet/ContextCommandletTest.java

```diff
@@ -0,0 +1,69 @@
+package com.devonfw.tools.ide.commandlet;
+
+import java.util.Locale;
+
+import org.junit.jupiter.api.Test;
+
+import com.devonfw.tools.ide.context.AbstractIdeContext;
+import com.devonfw.tools.ide.context.AbstractIdeContextTest;
+import com.devonfw.tools.ide.context.IdeContextConsole;
+
+public class ContextCommandletTest extends AbstractIdeContextTest {
+
+  /**
+   * Test of {@link ContextCommandlet} has name context.
+   */
+  @Test
+  public void testNameIsContext(){
+    //arrange
+    ContextCommandlet cxt = new ContextCommandlet();
+    //act & assert
+    assertThat(cxt.getName()).isEqualTo("context");
+  }
+
+  /**
+   * Test of {@link ContextCommandlet} does not require home.
+   */
+  @Test
+  public void testThatHomeIsNotReqired() {
+
+    // arrange
+    ContextCommandlet cxt = new ContextCommandlet();
+    //act & assert
+    assertThat(cxt.isIdeHomeRequired()).isEqualTo(false);
+  }
+
+  /**
+   * Test of {@link ContextCommandlet} getIdeContext.
+   */
+  @Test
+  public void testGetIdeContext() {
+
+    // arrange
+    ContextCommandlet cxt = new ContextCommandlet();
+    cxt.run();
```

Comment:
this would be `act` so move the comment up ;)

Reply:
As I removed this test (see next command) outdated and done.

Reply:
but should have noticed the wrong position of cxt.run() :)